### ISSUE TITLE
ModuleNotFoundError: No module named 'channel'

### ISSUE THREAD

**Issue by rajanpaul02-zz** [MBI-Div-B/sardana-AgilisAGAPMotorController#1]
Hi, I am trying to connect my Conex agap ag m100d, motorized mirror mount via python. I installed the pyagilis package still, I am receiving the error mentioned above in line 13.  Could you suggest what is going wrong here?
Thanks

**Comment by dschick** [comment]
Hi @rajanpaul02 ,

are you aware that this class is for using the Conex agap within the [Sardana](sardana-controls.org) framwork?
if so, please provide us with the complete error message and the Sardana version that you use.

If not, you should be fine with using the [pyagilis package](https://github.com/dschick/pyagilis) for pure python communication.

Best

Daniel

**Comment by rajanpaul02-zz** [comment]
Hi Daniel,
Thanks for the response! 
I installed Sardana 3.0.3, I wasn't aware though. Now I am, running only the pyagilis package. somehow I can't communicate with the com port. Kindly could tell me the procedure to install the pure python communication?

Thanks
Rajan

**Comment by dschick** [comment]
I think this is a pyvisa or pyserial issue then.
Please check that you have write access to the com-port, e.g. by `sudo chmod +x /dev/ttyUSB0` where `USB0` could be different for your system

**Comment by rajanpaul02-zz** [comment]
Hi Daniel,
I have pyserial installed. My comport is Com10. The problem is: it's creating a virtual serial port called AG-UC2-UC8 as Com9.
Could you let me know the exact procedure to communicate with the port, Com10 in my case?

Thanks
Rajan

**Comment by dschick** [comment]
hm, I think you need to use the virtual com port9.
Do you have any Conex Software installed?
If yes, which port is it using?

**Comment by rajanpaul02-zz** [comment]
Yes, the Conex agap software is installed and it's using the com10 port.

Thanks
Rajan

**Comment by rajanpaul02-zz** [comment]
Hi Daniel,
I couldn't establish the communication using the [pyagilis package](https://github.com/dschick/pyagilis). Could you tell me the procedure to install the communication using the package?
Thanks
Rajan

**Comment by dschick** [comment]
can provide any error messages? if it is about the pyagilis package, the discussion should be also moved there I would suggest.

**Comment by rajanpaul02-zz** [comment]
Yes, sure.
